src/Npgsql/PostgresMinimalDatabaseInfo.cs

```diff
@@ -83,6 +83,7 @@ static PostgresType[] CreateTypes(bool withMultiranges)
         Add(DataTypeNames.PgLsn, oid: 3220, arrayOid: 3221);
         Add(DataTypeNames.Unknown, oid: 705, arrayOid: 0);
         Add(DataTypeNames.Void, oid: 2278, arrayOid: 0);
+        Add(DataTypeNames.Cube, oid: 100000, arrayOid: 100001);
```

Comment:
We need to figure out a solution to this problem, this was required to make the tests pass, but probably creates many other unintended consequences.

Reply:
This shouldn't be necessary. The test suite allows you to install the extension, at which point the type info loading should be able to find it.

Reply:
We definitely shouldn't have cube in PostgresMinimalDatabaseInfo.